### DISCUSSION TITLE
Policies and Initiatives are not deployed

### DISCUSSION THREAD

**Discussion by rajiv-bansal** [Azure/data-management-zone#308] [Q&A]
Hi,

Infra/Policies folder contains definitions of custom policies, initiatives and assignment of initiative. But I don't see any code to deploy them. Are they being deployed. If yes, how?

Thanks

**Comment by marvinbuss** [comment]
Hi @rajiv-bansal.
You can deploy them to either a subscription scope or management group scope (recommended).
The template to deploy the definition to a management group scope can be found here: https://github.com/Azure/data-management-zone/blob/main/infra/Policies/PolicyDefinitions/deploy.policyDefinition.mg.json
The template to deploy the definition to a subscription scope can be found here: https://github.com/Azure/data-management-zone/blob/main/infra/Policies/PolicyDefinitions/deploy.policyDefinition.sub.json

Please mark this as answer, if this has answered your question. Thanks in advance! 👍

**Comment by rajiv-bansal** [comment]
Hi @marvinbuss ,

These templates deploy a single policy. But there are a lot of policies. Is there any template which iterates of the policies folders, deploy them, create an initiative from these policies and then assign the initiative.

Thanks,
Rajiv
